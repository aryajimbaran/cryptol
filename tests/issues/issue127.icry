:l issue127/issue127.cry