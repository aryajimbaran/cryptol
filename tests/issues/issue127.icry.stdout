Loading module Cryptol
Loading module Cryptol
Loading module Foo
Loading module issue127
